app/javascript/dashboard/components/widgets/conversation/linear/LinkIssue.vue

```diff
@@ -107,7 +110,11 @@ const onSearch = async value => {
       name: `${issue.identifier} ${issue.title}`,
     }));
   } catch (error) {
-    useAlert(t('INTEGRATION_SETTINGS.LINEAR.LINK.ERROR'));
+    const errorData = error.response.data;
+    const errorMessage =
+      errorData?.error?.errors?.[0]?.message ||
+      t('INTEGRATION_SETTINGS.LINEAR.LINK.ERROR');
+    useAlert(errorMessage);
```

Comment:
See if we use `parseAPIErrorResponse` from (store/utils/api.js), Or let us create a method for parsing the Linear API error so that we can reuse it for methods: `onSearch`, `linkIssue`, `loadLinkedIssue`, `unlinkIssue`.